### PR TITLE
Fix Vim plugin + update Flake references

### PR DESCRIPTION
**Pull request by toastal**
The Vim plugin was failing to build. I opened an upstream [merge request](https://github.com/NixOS/nixpkgs/pull/132517) in `nixpkgs` as well.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -70,8 +74,12 @@
               inherit (packages.${name}) version;
               name = "${name}-vim";
               src = self;
-              configurePhase = "cd vim/";
               buildInputs = [ packages.${name} ];
+              patchPhase = ''
```
Comment (soywod) [inline flake.nix:78]: Could you explain those lines? I mean, why they are necessary?

**Review by soywod — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (toastal) [reply flake.nix]: I understand now what happened so I did I different fix (the wrapper for building vim plugins upstream changed).

However, `''\n''` is lke starting a heredoc; it lets you write write a longer inline `bash` script in this block which if you have literally more than one line, is recommended.

**Comment by toastal** [comment]
I was pointed in the direction of the upstream Vim plugin building wrapper now by default disabling the configure phase with `dontConfigure = true;` being the default. This is really weird to me because there already was a check and the default is `configurePhase ? ""`.

**Comment by toastal** [comment]
Now, why Cachix is failing in some JS script is :shrug: to me. I don't know or want to know GitHub Actions. My upstream Vim plugin fix was merged and this builds for me locally.

**Comment by soywod** [comment]
Thanks :tada:

**Pull request merged**